p2p/peer.py

```diff
@@ -856,7 +854,9 @@ def _add_peer(self,
     async def _run(self) -> None:
         # FIXME: PeerPool should probably no longer be a BaseService, but for now we're keeping it
         # so in order to ensure we cancel all peers when we terminate.
-        self.run_task(self._periodically_report_stats())
+        if self.event_bus is not None:
+            self.run_daemon_task(self.handle_peer_count_requests())
+        self.run_daemon_task(self._periodically_report_stats())
```

Comment:
This is potentially not a good idea.  It'll cause the node to crash if peer stats reporting crashes.